Rise-Vision/rise-video
# Feature/integrate videojs

## Description
Improve integration of 3rd party dependencies with component.

## Motivation and Context
Some small changes to the way the js and css for videojs and videojs-playlist are integrated into the component.

While ideally we would simply reference the npm packages via `import` or some other method, this turns out to be non-trivial, and so I elected to do something simple which meets the main requirements of importing the 3rd party dependencies and also minimizing the need for template designers to add additional code.

I wrote up a [document with my reasoning here](https://docs.google.com/document/d/1qfoQa9EiVN9HpBeq6-LqoVSpDkMFPW7nBjpzraQw5i0/edit?usp=sharing), please feel free to comment if you can think of any more elegant methods than what I've done here.

## How Has This Been Tested?

- Integration tests have been added for new code. These are minimal, just quick checks to make sure that videojs and videojs-playlist are successfully included in the bundle.
- Changes have been tested on both Electron and Chrome extension players using [this presentation](https://apps.risevision.com/templates/edit/7900dcc5-72c0-46e1-b67f-29d96bc578a0/?cid=7fa5ee92-7deb-450b-a8d5-e5ed648c575f) which uses a test template pointed at a build of this version of `rise-video`. At Santiago's request I also added a `<rise-text>` element to the test template to ensure there we no negative interactions with other `rise-xxx` components.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
  - Manual Test: Completed as noted above
  - Automated Test: Completed as noted above
  - Monitoring: All requirements met
  - Rollback: The previous release has been tagged so rollback should be simple
  - Documentation: No documentation updates needed
  - Internal Communication: Support does not need to be notified as the component is not in use
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No release checklist items have been skipped.
